app/src/main/java/org/jellyfin/androidtv/ui/playback/PlaybackController.java

```diff
@@ -1048,7 +1048,9 @@ private void itemComplete() {
         if (nextItem != null) {
             Timber.d("Moving to next queue item. Index: " + (mCurrentIndex + 1));
 
-            if (userPreferences.getValue().get(UserPreferences.Companion.getNextUpEnabled())) {
+            BaseItemDto curItem = getCurrentlyPlayingItem();
```

Comment:
It looks like `getCurrentlyPlayingItem()` can return `null` so we need to handle that properly.

Reply:
I believe this is now resolved.